### PR TITLE
Added Logos to styleguide

### PR DESCRIPTION
**Pull request by slj**
Connects #1350

### CODE REVIEW

File: client/app/containers/StyleGuide/StyleGuideLogos.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        integration, dynamism, transparency, intuitiveness, and modernism.
+        Each Caseflow application has it’s own unique accent color but the
+        base color remains the same, gray-lighter.
+      </p>
+      <p>
+        When refering to the whole Caseflow system, you should use the
+        non-colored Caseflow logo.
+      </p>
+      <p className="logo-example">
+        <Logo app="efolder" /><span>e-Folder Express</span>
```
Comment (shanear) [inline client/app/containers/StyleGuide/StyleGuideLogos.jsx]: I think this should be "eFolder Express", @lakohl?

File: app/assets/stylesheets/_style_guide.scss [hunk 2]
```diff
@@ -146,3 +146,9 @@ pre {
     }
   }
 }
+
+.logo-example {
```
Comment (shanear) [inline app/assets/stylesheets/_style_guide.scss]: Do we need this? Shouldn't we be able to use `<b>` or something?

Comment (slj) [inline app/assets/stylesheets/_style_guide.scss]: I thought the `<b>` tag was being deprecated.  Didn't realize HTML5 still cares.  I'll definitely change this.

Comment (shanear) [inline app/assets/stylesheets/_style_guide.scss]: Maybe check USWDS. I know we shouldn't have to create styleguide specific css for this.

File: client/app/containers/StyleGuide/StyleGuideLogos.jsx (hunk 1 above)
Comment (slj) [inline client/app/containers/StyleGuide/StyleGuideLogos.jsx]: @abbyraskin told me it's **eFolder Express** - no hypen.

File: app/assets/stylesheets/_style_guide.scss (hunk 2 above)
Comment (slj) [inline app/assets/stylesheets/_style_guide.scss]: @gnakm does this look good to you?

screen shot 2017-04-24 at 4 37 58 pm

Comment (gnakm) [inline app/assets/stylesheets/_style_guide.scss]: 😍 lgtm

**Pull request merged**